app/services/passporting/means_passporter.rb

```diff
@@ -0,0 +1,23 @@
+module Passporting
+  class MeansPassporter < BasePassporter
+    def call
+      means_passport = []
+      means_passport << MeansPassportType::ON_AGE_UNDER18   if applicant_under18?
+      means_passport << MeansPassportType::ON_BENEFIT_CHECK if benefit_check_passed?
+
+      crime_application.update(means_passport:)
+
+      passported?
+    end
+
+    def passported?
+      crime_application.means_passport.any?
+    end
+
+    private
+
+    def benefit_check_passed?
+      !!applicant.passporting_benefit
```

Comment:
Is this doing the same as what `.present?` would do? - would that be more readable?

Reply:
Yep, it is really the same. I will change this in the follow-up PR. I agree it seems more readable on booleans 👍 
On strings however is not the same:

```ruby
> !!""
=> true
> "".present?
=> false
```

Reply:
Good to know, thank you!